Fix OpenGL ES 2 CMake variable names

FindOpenGLES2.cmake sets OPENGLES2_FOUND / OPENGLES2_INCLUDE_DIR /
OPENGLES2_gl_LIBRARY but CMakeLists.txt was referencing the same
variables without the "2" suffix, causing the GLES2 path to always
fail.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,10 +39,10 @@
 ENDIF(OPENGL_TYPE STREQUAL "AUTO")

 IF(OPENGL_TYPE STREQUAL "AUTO")
-  IF(OPENGLES_FOUND)
+  IF(OPENGLES2_FOUND)
     MESSAGE(STATUS "OpenGL ES 2 found; using that.")
     SET(OPENGL_TYPE "OPENGLES2")
-  ENDIF(OPENGLES_FOUND)
+  ENDIF(OPENGLES2_FOUND)
 ENDIF(OPENGL_TYPE STREQUAL "AUTO")

 IF(OPENGL_TYPE STREQUAL "AUTO")
@@ -56,9 +56,9 @@
 ENDIF(OPENGL_TYPE STREQUAL "OPENGL")

 IF(OPENGL_TYPE STREQUAL "OPENGLES2")
-  IF(NOT OPENGLES_FOUND)
+  IF(NOT OPENGLES2_FOUND)
     MESSAGE(FATAL_ERROR "OpenGL ES was requested but CMake couldn't find it.")
-  ENDIF(NOT OPENGLES_FOUND)
+  ENDIF(NOT OPENGLES2_FOUND)
 ENDIF(OPENGL_TYPE STREQUAL "OPENGLES2")

 # OpenAL
@@ -314,7 +314,7 @@

 IF(OPENGL_TYPE STREQUAL "OPENGLES2")
   ADD_DEFINITIONS(-DUSE_OPENGL_ES=2)
-  INCLUDE_DIRECTORIES(${OPENGLES_INCLUDE_DIR})
+  INCLUDE_DIRECTORIES(${OPENGLES2_INCLUDE_DIR})
   INCLUDE_DIRECTORIES(${EGL_INCLUDE_DIR})
 ENDIF(OPENGL_TYPE STREQUAL "OPENGLES2")

@@ -327,7 +327,7 @@
 ENDIF(OPENGL_TYPE STREQUAL "OPENGL")

 IF(OPENGL_TYPE STREQUAL "OPENGLES2")
-  TARGET_LINK_LIBRARIES(avp ${OPENGLES_gl_LIBRARY})
+  TARGET_LINK_LIBRARIES(avp ${OPENGLES2_gl_LIBRARY})
   TARGET_LINK_LIBRARIES(avp ${EGL_LIBRARIES})
 ENDIF(OPENGL_TYPE STREQUAL "OPENGLES2")

